src/main/java/core/basesyntax/FigurePainter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigurePainter {
```

Comment:
Not only Figure can have this behavior - remember about Abstraction
```suggestion
public interface Drawable {
```